Shared/PostCollection/CollectionListModel.swift

```diff
@@ -19,7 +19,8 @@ class CollectionListModel: NSObject, ObservableObject {
             try collectionsController.performFetch()
             list = collectionsController.fetchedObjects ?? []
         } catch {
-            print("Failed to fetch collections!")
+            // FIXME: Errors cannot be thrown out of the CollectionListView property initializer
+            fatalError(LocalStoreError.couldNotFetchCollections.localizedDescription)
```

Comment:
In a separate PR, I'll investigate refactoring this logic for fetching collections out of the CollectionListModel and into a `@FetchRequest` property on the CollectionListView, so we can better handle errors.